grpc/Cargo.toml

```diff
@@ -6,19 +6,33 @@ authors = ["gRPC Authors"]
 license = "MIT"
 
 [dependencies]
```

Comment:
In a follow up we should relax the versions required here unless we need a feature in a specific patch.